src/static/less/pages/for-filers.less

```diff
@@ -0,0 +1,161 @@
+// Mobile first!
+
+.for-filers {
+
+  .hero {
+    h1 {
+      padding-bottom: 5px;
+      background: transparent url() scroll no-repeat left center;
+      .avenir-next-bold;
+      .at2x( '../img/book-isocon.png', 42px, 42px );
```

Comment:
For now I'm removing this from these pages, I don't have those skills. :smile: 
But, I'll talk to @benguhin about it. He gave me that one.